src/cljc/quil/core.cljc

```diff
@@ -3362,6 +3368,33 @@
   (let [mode (u/resolve-constant-key mode rect-modes)]
     (.rectMode (current-graphics) mode)))
 
+(defn
+  ^{:requires-bindings true
+    :processing-name nil
+    :p5js-name nil
+    :category "Debugging"
+    :added "3.1.0"}
+  print-first-n
+  "Prints the provided arguments for the first `n` iterations."
+  [n & more]
+  (when (<= n (frame-count))
+    (apply print more)))
```

Comment:
Why `print` and not `println`? Will it print everything on single line?

Reply:
Thanks for spotting this, should be `println`. What happened was I only tested in the quil editor in the browser. Won't happen again.

I have now run the snippets for both clj and cljs from the quil repo and the behaviour is as expected after uncovering the following bugs:
* bug in `print-first-n` in the condition of the `when` (this was something I had fixed in the editor but forgot to also fix in the quil repo)
* bug in `print-every-n`: missed the NPE thrown when evaluating something like `(- 3 nil)` in clj (did not detect it in the editor because its not thrown in cljs)